src/test/java/uk/nhs/tis/trainee/actions/service/ActionServiceTest.java

```diff
@@ -117,6 +122,7 @@ void shouldNotInsertReviewDataActionOnFirstSightOfPastProgrammeMembership() {
 
     assertThat("Unexpected action count.", actions.size(), is(0));
     verifyNoInteractions(repository);
+    verifyNoMoreInteractions(eventPublishingService);
```

Comment:
Use `verifyNoInteractions` instead of `verifyNoMoreInterations` for clarity (several other instances need updating too)